src/components/hooks/useActiveRedeemablePolicies.js

```diff
@@ -0,0 +1,14 @@
+import { useQuery } from '@tanstack/react-query';
+import { useContext } from 'react';
+import { AppContext } from '@edx/frontend-platform/react';
+import useEnterpriseCustomer from './useEnterpriseCustomer';
+import { queryRedeemablePolicies } from '../app/data';
+
+export default function useActiveRedeemablePolicies() {
+  const { authenticatedUser: { userId: lmsUserId } } = useContext(AppContext);
+  const { uuid: enterpriseUuid } = useEnterpriseCustomer();
+  const { data } = useQuery(queryRedeemablePolicies({ enterpriseUuid, lmsUserId }));
+  return {
+    activeRedeemablePolicies: data.redeemablePolicies.filter(policy => policy.active),
```

Comment:
Note: any policies returned by the `credits_available` API are guaranteed to already be `active`; if it wasn't, it wouldn't be returned by the API :) In my draft PR, I renamed this to `useRedeemablePolicies`.